### PR TITLE
TripletLoss example

### LINKED ISSUE #334: Request for example: Triplet Loss
I think its worth creating issues for key examples that we want shown in Addons. I plan on working on this one in the next day or so.

### PR DESCRIPTION
**Pull request by seanpmorgan**
Closes #334

### CODE REVIEW

File: examples/losses_triplet.ipynb [hunk 1]
```diff
@@ -0,0 +1,374 @@
+{
```
Comment (WindQAQ) [inline examples/losses_triplet.ipynb]: <p>nit: the new tfa loss</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (WindQAQ) [inline examples/losses_triplet.ipynb]: <p>The clusters look great! Maybe showing the class for each cluster would be much better!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

**Comment by WindQAQ** [comment]
Hmm, seems ReviewNB is probably not a great tool to show the comments in GitHub...

File: examples/losses_triplet.ipynb (hunk 1 above)
Comment (seanpmorgan) [reply examples/losses_triplet.ipynb]: <p>Good catch thanks! I've removed the line</p>

Comment (seanpmorgan) [reply examples/losses_triplet.ipynb]: <p>Good idea. It's a bit difficult to do in the projector. Let me know if the updated image works.</p>

**Comment by seanpmorgan** [comment]
> Hmm, seems ReviewNB is probably not a great tool to show the comments in GitHub...

Yeah not great, but I just read comments in the ReviewNB webpage. Please check out the changes when you have time

**Pull request merged**